### ISSUE TITLE
Cucumber does not run scenario with forward slash ( / )

### ISSUE THREAD

**Issue by rewgoes** [cucumber/cucumber-android#80]
If I have a Scenario Outline's name that contains a forward slash ( / ), cucumber stops running and does not run any other steps after that.

Example:
```gherkin
Scenario Outline: This is a scenario/example using forward slash
```

**Version:**
io.cucumber:cucumber-android:4.8.4

**Comment by lsuski** [comment]
You can't have any character which is not file name compatible.

**Comment by rewgoes** [comment]
@lsuski , that is it? No chances of fixing it or supporting it?

**Comment by lsuski** [comment]
There is an option in @CucumberOptions useFileNameCompatibleName - you can try it but I'm not sure if this is respected in cucumber-android. You can also provide a PR with fix.I think that changing scenario name to not contain / is just simpler

**Comment by rewgoes** [comment]
@lsuski, after a few tests, I've noticed that this only happens if I enable `ANDROIDX_TEST_ORCHESTRATOR`.

The sample project in this repo, cukeulator, has the AndroidX Orchestrator disabled by default. As soon as you enable it, tests with characters that are not file name compatible simply does not execute (stops the runner).

I've tried passing `useFileNameCompatibleName=false` to `@CucumberOptions` but it didn't make any difference. The default value is already false.
